postpic/datahandling.py

```diff
@@ -63,6 +63,15 @@ def __init__(self, name='', unit=''):
         self._grid_node = np.array([])
         self._linear = None
 
+    def __copy__(self):
+        '''
+        returns a shallow copy of the object.
+        This method is called by `copy.copy(obj)`.
+        '''
+        ret = type(self)()
```

Comment:
copy shouldn't call `__init__`, see https://stackoverflow.com/questions/1500718/what-is-the-right-way-to-override-the-copy-deepcopy-operations-on-an-object-in-p#comment21452843_1500887. Better use:

`cls = type(self)`
`ret = cls.__new__(cls)`